tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -122,3 +122,46 @@ Feature: spaces management
       | David | viewer  |
       | Edith | viewer  |
     And "Alice" logs out
+
+
+  Scenario: admin user can manage the spaces created by other space admin user
```

Comment:
can't we combine this scenario with above `Scenario: spaces can be managed in the admin settings via the context menu` so that we don't have to repeat same step multiple run and can cover more tests case

Reply:
The above scenario can be covered only from the batch action. No need to combine since combining both can be expensive as once using batch action and next time we need to use the context menu the context menu works same as batch action. Also viktor confirmed that this test case can be covered with batch action only.